libflux/src/flux/semantic/builtins.rs

```diff
@@ -301,6 +301,178 @@ pub fn builtins() -> Builtins<'static> {
+                    forall [t0, t1] where t0: Row, t1: Row (
+                        <-tables: [t0],
+                        ?column: string
+                    ) -> [t1]
+                "#),
+                "drop" => Node::Builtin(r#"
+                    forall [t0, t1, t2] where t0: Row, t1: Row, t2: Row (
+                        <-tables: [t0],
+                        ?fn: (column: string) -> bool,
+                        ?columns: t1
+                    ) -> [t2]
+                "#),
+                "duplicate" => Node::Builtin(r#"
+                    forall [t0, t1] where t0: Row, t1: Row (
+                        <-tables: [t0],
+                        column: string,
+                        as: string
+                    ) -> [t1]
+                "#),
+                "duration" => Node::Builtin("forall [t0] (v: t0) -> duration"),
+                "elapsed" => Node::Builtin(r#"
+                    forall [t0, t1] where t0: Row, t1: Row (
+                        <-tables: [t0],
+                        ?unit: duration,
+                        ?timeColumn: string,
+                        ?columnName: string
+                    ) -> [t1]
+                "#),
+                "exponentialMovingAverage" => Node::Builtin(r#"
+                    forall [t0, t1] where t0: Numeric (
+                        <-tables: [{ _value: t0 | t1 }],
+                        n: int
+                    ) -> [{ _value: t0 | t1}]
+                "#),
+                "false" => Node::Builtin("forall [] () -> bool"),
+                "fill" => Node::Builtin(r#"
+                    forall [t0, t1, t2] where t0: Row, t2: Row (
+                        <-tables: [t0],
+                        ?column: string,
+                        value: [t1],
+                        usePrevious: bool
+                    ) -> [t2]
+                "#),
+                "filter" => Node::Builtin(r#"
+                    forall [t0] where t0: Row (
+                        <-tables: [t0],
+                        fn: (r: t0) -> bool
+                    ) -> [t0]
+                "#),
+                "first" => Node::Builtin(r#"
+                    forall [t0] where t0: Row (
+                        <-tables: [t0],
+                        ?column: string
+                    ) -> [t0]
+                "#),
+                "float" => Node::Builtin("forall [t0] (v: t0) -> float"),
+                "getColumn" => Node::Builtin(r#"
+                    forall [t0] where t0: Row (
+                        <-table: [t0],
+                        column: string
+                    ) -> t0
+                "#),
+                "getRecord" => Node::Builtin(r#"
+                    forall [t0, t1] where t0: Row, t1: Row (
+                        table: [t0],
+                        idx: int
+                    ) -> [t1]
+                "#),
+                "group" => Node::Builtin(r#"
+                    forall [t0, t1] where t0: Row (
```

Comment:
Can you remove the extra parameter `t1`?

Reply:
I wonder if this should fail in the parser. Like, the test for this should have failed, I think. Thoughts? Should I file a bug?

Reply:
You mean if an extra parameter is included but never referenced, it should fail to parse? That would be useful and an issue would be great.

Reply:
https://github.com/influxdata/flux/issues/2275